example/advection/parthinput.advection

```diff
@@ -83,6 +83,11 @@ file_type = hst
 dt = 0.05
 
 <parthenon/output4>
+file_type = hst
+dt = 0.1
+packages = advection_app
```

Comment:
Example for including only history output from specific package (`advection_app` has no enrolled history outputs so `advection.out4.hst` only has `t` and `dt` fields).

Reply:
`packages = ` seems to signal to me that I could supply a comma separated list of packages to be enrolled in output4.  Is that the case? Will keep reading...